src/main/scala/com/thirdship/libunit/units/MetricTSUnit.scala

```diff
@@ -0,0 +1,52 @@
+package com.thirdship.libunit.units
+
+import com.thirdship.libunit.utils.{ExactString, FuzzyString, WordString}
+import com.thirdship.libunit.utils.Helpers._
+import com.thirdship.libunit._
+
+class MetricTSUnit(unit: String, parseList: List[FuzzyString]) {
```

Comment:
I'm not sure I understand your suggestion. Could you clarify what that would look like?